engine/src/main/java/com/jadice/barcode/linear/c128/Code128Decoder.java

```diff
@@ -53,115 +54,115 @@
    * locations.
    */
   private static final int PATTERNS[] = {//
```

Comment:
The formatter messed up this part. We should consider using eclipse formatter tags.